core_tic_tac_toe/display.py

```diff
@@ -65,11 +65,14 @@ def __generate_rows(self, move):
 
         for idx, position in enumerate(move):
             if idx != len(move) - 1:
-                if type(position) == int and position // 10 > 0:
+                if type(position) == int and self.__is_double_digit(position):
```

Comment:
If you always want to allow a certain amount of space, you can probably use some additional notation from [Python's format string syntax](https://docs.python.org/3/library/string.html#format-string-syntax).

Reply:
Thank you for sending this over! I was able to find the correct formatting type.